### DISCUSSION TITLE
Flocking behavior on python code

### DISCUSSION THREAD

**Discussion by JoelJohera** [JoelForamitti/agentpy#63] [Q&A]
Hi,

I'm trying to run a simulation of the flocking behavior in PyCharm. My end goal, after I've checked that the simulation is to my liking, is to get the positions of all the points at all the frames.
Where can I find an example of the flocking behavior that works in PyCharm?

Thanks for your time.

**Comment by jofmi** [comment]
Hi @JoelJohera. I am not using PyCharm but I know that it has a plugin to support Jupyter Notebooks. With this, you should be able to run the flocking notebook from the documentation.

**Reply by JoelJohera** [reply]
I've tried running the flocking notebook from the documentation but I got to many errors that is why I was asking of running the agentpy in normal python outside a notebook.

**Reply by JoelJohera** [reply]
After some more testing, I managed to get it to work. Is there a way to export the location of each point at all frames? Like in a txt file for each point with a line for each frame?

Also, inner_radius, outer_radius, and border_distance, what are the units for these values? Centimeters, Meters, Kilometers?
